bin/apt-install.sh

```diff
@@ -49,7 +49,7 @@ for step in "${steps[@]}"; do
             ;;
         other-CI-tools)
             # Assumes other steps have been run before
-            apt-get install -q -y clang-format-$LLVM_VERSION parallel pip shellcheck
+            apt-get install -q -y clang-format-$LLVM_VERSION parallel pip rsync shellcheck
```

Comment:
This step is used to build docker images for our CI, if `rsync` is not used in the CI, I suggest you put this another step (`other-dev-tools`?)